srcpkgs/singularity/template

```diff
@@ -0,0 +1,38 @@
+# Template file for 'singularity'
+pkgname=singularity
+version=3.2.1
+revision=1
+archs="~arm* ~aarch*"
+build_style=configure
+hostmakedepends="go git pkg-config"
+makedepends="gpgme-devel libressl-devel libuuid-devel libseccomp-devel"
+depends="squashfs-tools"
+short_desc="Open source container platform designed to be simple, fast, and secure"
+maintainer="Emilio Palumbo <emiliopalumbo@gmail.com>"
+license="BSD-3-Clause"
+homepage="https://www.sylabs.io/singularity"
+distfiles="https://github.com/sylabs/singularity/releases/download/v${version}/singularity-${version}.tar.gz"
+checksum=d4388fb5f7e0083f0c344354c9ad3b5b823e2f3f27980e56efa7785140c9b616
+nopie=yes
+
```

Comment:
Please remove that empty line and sort variables, see xlint source for correct order.